core/src/main/java/com/sequenceiq/cloudbreak/service/cluster/ClusterRepairService.java

```diff
@@ -342,7 +351,7 @@ private RepairValidation validateSelectedNodes(Stack stack, Map<HostGroupName, S
         if (reattach) {
             for (InstanceMetaData instanceMetaData : instances) {
                 validationResult.addAll(validateOnGateway(stack, instanceMetaData));
-                if (hasEphemeralStorage(stack, hostGroupName.value())) {
+                if (hasNotStoppableEphemeralStorage(stack, hostGroupName.value())) {
```

Comment:
why is this in the for loop with instances?

Reply:
good point, thanks, moved it outside